docker/release.dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:16.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG HOME=/root
+ENV PATH=$HOME/go/bin:$PATH
+
+WORKDIR $HOME
+
+# install base dependencies
+
+RUN apt-get update && apt-get install -y software-properties-common && add-apt-repository ppa:ubuntu-toolchain-r/test && add-apt-repository ppa:deadsnakes/ppa
+RUN apt-get update \
+    && apt-get install -y git curl wget gcc-9 g++-9 build-essential patchelf make libssl-dev zlib1g-dev \
+    libbz2-dev libreadline-dev libsqlite3-dev llvm \
+    libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev \
+    python3.6 python3.7 python3.8 python3.9 python3.10 \
+    python3.6-dev python3.7-dev python3.8-dev python3.9-dev python3.10-dev \
```

Comment:
`python3.6` is dropped right?

Reply:
yep but it doesn't matter to build wheels